starter-code/css/main.css

```diff
@@ -0,0 +1,113 @@
+body{
+    font-family: Lato, sans-serif;
+    padding-left: 30px;
+    margin-left: 20px;
+    border-left-style: solid;
+    border-left-color: rgba(195,195,195,.8);
+
```

Comment:
Typically when you are specifying border CSS, you want to set the width, style, and color. You can do that in one line: `border-left: 10px solid rgba(195, 195, 195, .8);`